controllers/doctors_HISP.py

```diff
@@ -1,7 +1,51 @@
-﻿from bottle import route, view, request, response
+﻿from bottle import route, view, request, response, template
 from datetime import datetime
 import sqlite3
 from permissions import permissions
+import requests, json
+
+def create_patient(url,patient):
+    # EC 112115 Expects patient dictionary and HISP address
+    # returns response to parse out id
+    query = 'base/Patient/'
+    base_url = url + query
+    # create the payload using example
+    payload = {"active": 'true',
+        "address": [{"city": "",
+        "line": [""],
+        "postalCode": "",
+        "state": "",
+        "use": ""}],
+        "birthDate": "",
+        "gender": "",
+        "id": "",
+        "name": [{"family": [""], "given": [""]}],
+        "resourceType": "",
+        "text": {"status": ""}}
+
+    payload["name"][0]["family"] = patient['lname']
+    payload["name"][0]["given"] = patient['fname']
+    payload["birthDate"] = patient['dob']
+    payload["active"] = 'true'
+    payload["resourceType"] = "Patient"
+    payload["text"] = {"status": "generated"}
+    payload["id"] = '1'
+    payload["gender"] = patient['sex']
```

Comment:
@omtinez I don't think we need this for the demo. Let's hold off on this pull request.